deploy_all.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+
+NETWORKS=fuji,mumbai,goerli,avalanche,polygon,mainnet
+
+yarn codegen
+
+# Use comma as separator and apply as pattern
+for network in ${NETWORKS//,/ }
```

Comment:
given that you're using bash instead of plain ol' posix shell, you could define variable as an array like so

`NETWORKS=(fuji mumbai goerli)`

and then iterate through the array with `for network in "${NETWORKS[@]} ; `

Although the preferred way would be to use a posix shell (`#!/bin/sh`) and separate network names with spaces, and let the for loop split on spaces like it does by default

```
#!/bin/sh

memes="meme1 meme2 meme3"

for i in ${memes} ; do
  echo "$i"
done
```